content/spin/designing-spin-apps.md

```diff
@@ -0,0 +1,44 @@
+title = "Designing Spin Applications"
+template = "spin_main"
+date = "2023-05-05T00:01:01Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/designing-spin-apps.md"
+
+---
+
+- [Built With Spin](#built-with-spin)
+- [Next Steps](#next-steps)
+
+## Built With Spin
+
+Spin can be used to build many different types of applications. The following blog articles show how different applications are built with Spin:
+
+[How I Built a Like Button for My Blog with Spin](https://www.fermyon.com/blog/how-i-built-a-like-button-for-my-blog-with-spin)
+
+The 'Building a Social App with Spin' series covers the process and decision-making of building an authenticated, dynamic, database-backed application and API, right the way from downloading Spin and setting up the project to deploying the finished application to the cloud.
+
+{{ details "Building a Social App With Spin" "- [Part 1: Project Setup and first API](https://www.fermyon.com/blog/building-a-social-app-with-spin-1)\n- [Part 2: Vue.js App and Token Verification](https://www.fermyon.com/blog/building-a-social-app-with-spin-2)\n- [Part 3: Post API and Spin SDK Bindings](https://www.fermyon.com/blog/building-a-social-app-with-spin-3)\n- [Part 4: Go Postgres Usage](https://www.fermyon.com/blog/building-a-social-app-with-spin-3-5)\n- [Part 5: Key-Value Storage and Fermyon Cloud](https://www.fermyon.com/blog/building-a-social-app-with-spin-4)"}}
```

Comment:
The `details` code results in tucking the links away inside an expander box.

![image](https://user-images.githubusercontent.com/865538/236994264-8d64bbab-a10b-448e-805a-312e1a53075f.png)

To me, and I know this is personal taste, it looks much better when you can see the series:

![image](https://user-images.githubusercontent.com/865538/236994686-aa6eafda-173c-42fd-ab5f-85ba49afd98b.png)

```suggestion
* [Part 1: Project Setup and first API](https://www.fermyon.com/blog/building-a-social-app-with-spin-1)
* [Part 2: Vue.js App and Token Verification](https://www.fermyon.com/blog/building-a-social-app-with-spin-2)
* [Part 3: Post API and Spin SDK Bindings](https://www.fermyon.com/blog/building-a-social-app-with-spin-3)
* [Part 3.5: Go Postgres Usage](https://www.fermyon.com/blog/building-a-social-app-with-spin-3-5)
* [Part 4: Key-Value Storage and Fermyon Cloud](https://www.fermyon.com/blog/building-a-social-app-with-spin-4)"
```

(In the suggestion, though not the screenshot, I've retained the original blog post series numbering.  My reasoning for this was it felt confusing clicking through on a 5-part series and seeing "n/4" or clicking on part 5 and seeing part 4.  As always, matter of taste.)

Reply:
Sure thing, I understand (the part 3.5 had me also)